src/bin/bitmaskd.rs

```diff
@@ -579,12 +597,19 @@ async fn main() -> Result<()> {
         .route("/carbonado/:pk/:name", post(co_store))
         .route("/carbonado/:pk/:name/force", post(co_force_store))
         .route("/carbonado/:pk/:name/metadata", get(co_metadata))
-        .route("/carbonado/:pk/:name", get(co_retrieve))
-        .layer(CorsLayer::permissive());
+        .route("/carbonado/:pk/:name", get(co_retrieve));
 
     let network = get_network().await;
     switch_network(&network).await?;
 
+    #[cfg(not(target_arch = "wasm32"))]
```

Comment:
I'm not sure this block makes sense, since this file will never be compiled to wasm